test/distributed/test_c10d_nccl.py

```diff
@@ -3064,6 +3064,8 @@ def setUp(self):
         os.environ["NCCL_ALGO"] = "NVLS"
         os.environ["NCCL_DEBUG"] = "INFO"
         os.environ["NCCL_DEBUG_SUBSYS"] = "NVLS"
```

Comment:
Do we not need to test both debug subsys?

Reply:
Not both. In 2.24.3, NCCL moved around the debug output I'm interested in from NVLS sub system to REG sub system. That's why just updating the NCCL_DEBUG_SUBSYS if we encounter NCCL 2.24.3+.